### DISCUSSION TITLE
[Bug] Lava Absorbant Sponge doesn't work

### DISCUSSION THREAD

**Discussion by grechnevi** [Pryaxis/TShock#1848] [Q&A] [answered]
[Lava Absorbant Sponge](https://terraria.gamepedia.com/Lava_Absorbant_Sponge) doesn't work.
Text in console logs:
> Player Username has been disabled for Spreading lava without holding a lava bucket.

**Comment by QuiCM** [comment]
Thanks, I have raised https://github.com/Pryaxis/TShock/issues/1852 to track this

**Reply by grechnevi** [reply]
[Bottomless Lava Bucket](https://terraria.gamepedia.com/Bottomless_Lava_Bucket) doesn't work either.
Text in console logs:
> Player Username has been disabled for Spreading lava without holding a lava bucket.

**Comment by hakusaro** [comment, marked as answer]
This should be fixed in the upcoming/next pre-release as #1852 is now closed.
